asio/include/asio/detail/impl/posix_thread.ipp

```diff
@@ -3,6 +3,7 @@
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 //
 // Copyright (c) 2003-2023 Christopher M. Kohlhoff (chris at kohlhoff dot com)
+// Copyright (c) 2023, BlackBerry Limited. All Rights Reserved.
```

Comment:
"All Rights Reserved" isn't going to fly with the Boost Software License.

Reply:
Done